### ISSUE TITLE
'Something is wrong' message with No Internet notification for Android Pie+

### ISSUE THREAD

**Issue by pureskar** [mgks/Android-SmartWebView#98]
My app is running awesome in every other phones but not Android Pie. Need to know how to update the current app to Android Pie?

**Comment by mgks** [comment]
I've tested it recently, it's working fine on my Pie device. Can you elaborate what's actually happening on launching the app? If you can share log of failure.

**Comment by pureskar** [comment]
Where can i find this log? Also, i have been testing it out on another person's phone who would be using the App. Here is the screenshot of the error:
![IMG-20190710-WA0001](https://user-images.githubusercontent.com/42010259/61298363-b2ce3880-a7fb-11e9-837b-4d6ba869a49b.jpg)

Here you can see at the top the internet connectivity is running in Wifi. But still the problem persists. But this is not the case for any phone below Pie.

**Comment by mgks** [comment]
I thought app is going down but internet error can be caused by number of reasons, you can get project log from Android Studio Log and without any log I can't help you any further.

**Comment by pureskar** [comment]
I understand but the app isnt producing any errror logs during the build. Also this thing is occuring in all Pie phones. I have an Oreo phone. Hence no error is being shown on my end. But the moment the app runs in Pie, it shows this above error. I looked into the log. There is nothing there. As i said, this is happening on sets with Pie, not below.

**Comment by mgks** [comment]
> this is happening on sets with Pie, not below.

so you tested it on multiple pie devices?

**Comment by pureskar** [comment]
Yes. Did it on 4 Pie devices. And 1 Oreo & 2 Nougats. Only the Pie ones had the prob.

**Comment by mgks** [comment]
I'll be testing it soon. will let you know.

**Comment by pureskar** [comment]
Sure. Thanks. Do let me know if you require the Gradle Files as well.

**Comment by ariqbanihardi** [comment]
Hi, is that any suggest about this? @mgks  it happen to me too. 
This is the verbose Logs that I get :
`/mgks.os.webview I/chromium: [INFO:CONSOLE(2)] "jQuery.Deferred exception: Cannot read property 'split' of undefined", source: file:///android_asset/jquery.js `

**Comment by mgks** [comment]
Android Studio is loading source files into the build and while reading `jquery.js` file it found a property that build can't understand.
First keep your Android studio and dependencies updated. If the error still persists, you can remove the `JS` file from specific folder, it's just for the demo file, it doesn't impact the actual program.

**Comment by ariqbanihardi** [comment]
I'm not using this offline source file stuff. I just change the ASWV_URL to some http address. It's work for Https address, but when I addresses it to our web server which is Http only URL is not work. then the error like @pureskar error appeared.

**Comment by junedkh** [comment]
I cant add my apache server address please Help

**Comment by mgks** [comment]
@junedkh are you trying to run your localhost in the app? I'm not able to understand what you mean.

**Comment by junedkh** [comment]
Ignore bro
Now I am improved
